input/pagecontent/general-guidance.md

```diff
@@ -110,6 +110,265 @@ Example: Patient resource with interpreter required and language is known
+* Note: in this case laterality is not expressed in coded form 
+
+Example Condition resource cellulitis of knee, laterality as text only
+~~~
+{
+  "resourceType" : "Condition",
+  "id" : "cellulitis",
+  "clinicalStatus" : {
+    "coding" : [
+      {
+        "system" : "http://terminology.hl7.org/CodeSystem/condition-clinical",
+        "code" : "active"
+      }
+    ]
+  },
+  "verificationStatus" : {
+    "coding" : [
+      {
+        "system" : "http://terminology.hl7.org/CodeSystem/condition-ver-status",
+        "code" : "confirmed"
+      }
+    ]
+  },
+  "category" : [
+    {
+      "coding" : [
+        {
+          "system" : "http://terminology.hl7.org/CodeSystem/condition-category",
+          "code" : "encounter-diagnosis",
+          "display" : "Encounter Diagnosis"
+        }
+      ]
+    }
+  ],
+  "code" : {
+    "coding" : [
+      {
+        "system" : "http://snomed.info/sct",
+        "code" : "13301002",
+        "display" : "Cellulitis of knee"
+      },
+      "text" : "Cellulitis of knee, Right"
+    ]
+  }
+}
+~~~
+
+3\. Primary `code` and `bodySite` with laterality coded separately.
+* For systems that have 
+  * Pre-coordinated coding describing primary concept without body site.
+  * Body site with laterality is recorded as coded value.
+* The `code` element is used:
+  * Contains `coding` for primary concept including body site without laterality.
+  * Use `text` to describe concept fully, this can include information on recorded body site and laterality as text.
+* MAY record coded `bodySite`, optional when this element is not *Must Support* in AU Core profiles.
+
+Example Condition resource cellulitis, body site right knee
+~~~
+{
+  "resourceType" : "Condition",
+  "id" : "cellulitis",
+  "clinicalStatus" : {
+    "coding" : [
+      {
+        "system" : "http://terminology.hl7.org/CodeSystem/condition-clinical",
+        "code" : "active"
+      }
+    ]
+  },
+  "verificationStatus" : {
+    "coding" : [
+      {
+        "system" : "http://terminology.hl7.org/CodeSystem/condition-ver-status",
+        "code" : "confirmed"
+      }
+    ]
+  },
+  "category" : [
+    {
+      "coding" : [
+        {
+          "system" : "http://terminology.hl7.org/CodeSystem/condition-category",
+          "code" : "encounter-diagnosis",
+          "display" : "Encounter Diagnosis"
+        }
+      ]
+    }
+  ],
+  "code" : {
+    "coding" : [
+      {
+        "system" : "http://snomed.info/sct",
+        "code" : "128045006",
+        "display" : "Cellulitis"
+      },
+      "text" : "Cellulitis, Right Knee"
+    ]
+  },
+  "bodySite" : [
+    {
+      "coding" : [{
+        "system" : "http://snomed.info/sct",
+        "code" : "6757004",
+        "display" : "Structure of right knee region"
+      }],
+      "text" : "Right Knee"
+    }
+  ]
+
+}
+~~~
+
+
+4\. Primary `code` and `bodySite` without laterality coded separately and also separate laterality qualifier.
+* For systems that have 
```

Comment:
Reformat this to be more similar to Missing Data advice so it becomes:

* For systems that have pre-coordinated coding describing primary concept without body site and body site without laterality is as separate coded value, and laterality qualifier recorded separately e.g. left, right:
  * use the `code` element:
    * `code.coding` contains the primary concept alone (no body site or laterality).
    * `code.text` describes the concept fully, this can include information on recorded body site and laterality as text. 
  * optionally, coded element bodySite may be supplied containing:
     * `bodySite.coding` contains the coded body site without laterality.
     * `bodySite.text` describes the body site concept fully, this can include information on recorded laterality as text e.g. ', Right'.